src/pages/ProductsList/components/ImagesContainer.jsx

```diff
@@ -52,19 +115,27 @@ const TextContainer = styled.div`
   align-items: flex-start;
   margin-top: 10px;
   position: absolute;
-  margin-top: -86px;
+  margin-top: -85px;
   margin-left: 36px;
-  color: black;
+  color: #f5efe7;
 `;
 
-const Text1 = styled.div`
+const CampName = styled.div`
   ${boxSize(280, 30)}
   margin-top: -20px;
+  ${fontMix(16)}
 `;
 
-const Text2 = styled.div`
+const CampRegion = styled.div`
   ${boxSize(280, 30)}
   margin-top: -3px;
+  ${fontMix(14)}
+`;
+
+const CampPrice = styled.div`
+  ${boxSize(280, 30)}
+  margin-top: -8px;
+  ${fontMix(14)}
```

Comment:
```suggestion
  ${boxSize(280, 30)};
  ${fontMix(14)};
  margin-top: -8px;
```
mixin으로 사용하는 부분은 한 구역에 함께 배치해서 사용해주세요